### DISCUSSION TITLE
Limited Scan results

### DISCUSSION THREAD

**Discussion by Naveensparks** [projectdiscovery/httpx#491] [General] [answered]
I'm having a set of URLs in a text file , say urls.txt 
I used the command " cat urls.txt | httpx -status-code " which gives only few results. Even though the urls.txt file has many URLs the scanning isn't done I guess. 
What am I missing here ?

HTTPX version: v1.1.4
go version go1.17.5 linux/amd64

**Comment by ehsandeep** [comment, marked as answer]
@Naveensparks not sure type of data `urls.txt` contains, generally if passed through httpx, only alive URLs supposed to be printed back in results, let us know if you noticed any valid URL which is part of input but not output.
